### ISSUE TITLE
Proper files structure and common styling files.

### ISSUE THREAD

**Issue by accakks** [JessicaML/ScienceMotion-Static#33]

**Comment by accakks** [comment]
1) Every lesson can be put into folder of relevant subject instead of so many folders.
2) Common styling files.  Basic layout of lessons page is almost the same, so there can be a common css file which can be included again & again in every lesson code. Lesson specific styles can be in seperate file inside lesson folder.
3) Bootstrap can be included from CDN & that maybe included in instructions in README or a basic template of index.html file can be suggested in some common folder (like the styling one)

These things would make project look more arranged & clean for so many people to work together!

**Comment by JessicaML** [comment]
good idea Aakanksha!
1. Yes, for now 3 main folders for biology, chemistry, physics would be ideal, with topics and sub-topics lessons within them (see the lesson spreadsheet). https://docs.google.com/spreadsheets/d/1s-CqehOYRcrJmvR3d7WqAs7j7dDJLHuJlcRCXMLvwok/edit#gid=203115582

2. There is already a lesson_template.css file inside css/lesson_template.css
3. For bootstrap, the ideal is to have a CDN and also a local version to fall back on.
4. We should be minifying files (e.g. JQuery) where possible too.

**Comment by JessicaML** [comment]
Shall I assign this task to you? ^^

**Comment by accakks** [comment]
Yes surely! :)
But it might take a little time as my colleges are going on!
If there's no hurry, I'd love to fix this one. :+1:

**Comment by JessicaML** [comment]
:D Thanks Aakanksha!

**Comment by saloni-garg** [comment]
Yeah! @accakks that's indeed a great idea. Maybe I can help you with this.

**Comment by accakks** [comment]
Hi @saloni-garg  Sure! :D Do you have any more suggestions to this? 
We can discuss here & then start working on this together!

**Comment by JessicaML** [comment]
@accakks @saloni-garg Let's pause on this one - we are migrating the content to a rails/react app so we can wait til we are using that!

**Comment by accakks** [comment]
Sorry.  I forgot this one. And yes, we can work on this after that. :)

**Comment by bhawna-singh** [comment]
@JessicaML  can you please guide me and my partner @gargshruti30 a bit about the project as we would like to contribute towards the project ?!

**Comment by JessicaML** [comment]
Hello @bhawna-singh and @gargshruti30! Thanks for your interest! :D Shall we have a call so I can explain a little more about the project? Do you use Slack? You can join the slack channel here: https://join.slack.com/t/sciencemotions/shared_invite/enQtNTIyNDE4OTk1OTU1LThjYjRiMTBkNjkyNDQ1ZjZhNGEwNDFhNTE1NGZiZDllZDMwY2M4NTIyYzdhYTMwOGI3YTQ0MGJhMTFhM2YwNmU

**Comment by gargshruti30** [comment]
@JessicaML I have joined the slack channel. I look forward to discussing this project with you.

**Comment by bhawna-singh** [comment]
We've joined the slack channel and we are highly inquisitive for the
project , up for the discussion through a call or any other medium.
